kotlinx-coroutines-core/common/src/flow/Flow.kt

```diff
@@ -195,7 +194,6 @@ public interface Flow<out T> {
  * }
  * ```
  */
-@FlowPreview
 public abstract class AbstractFlow<T> : Flow<T> {
```

Comment:
I'm still not satisfied by `collectSafely` abstract method here that is becoming public with this change. It's name name has some non-trivial connotations. I suggest to leave it in preview. The safe `flow { ... }` builder does not internally use this class, so it has a stable ABI. We don't _have_ to make it public on the first release.

Reply:
Good point